terraform/layers/base/backend.tf

```diff
@@ -1,3 +1,8 @@
 terraform {
-  backend "s3" {}
+  backend "s3" {
```

Comment:
We should add these configs to the .tfbackend files under the environment folder rather than inside the layers to allow different environments to run the same code

Reply:
Have moved the config references to the .tfbackend files